src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -104,8 +110,13 @@ class CLIJsonWebKeyOperation(str, Enum):
                                                          'Vnet/subnet pairs or subnet resource ids.')
 
     with self.argument_context('keyvault update') as c:
-        c.argument('enable_soft_delete', arg_type=get_three_state_flag(),
-                   deprecate_info=c.deprecate(expiration='2.11.0', hide='2.11.0'))
+        c.argument('enable_soft_delete', arg_type=get_three_state_flag(), deprecate_info=Deprecated(
```

Comment:
Why not use `deprecate_info=c.deprecate(message_func=)`?

Reply:
Good point, it works!
Actually I decided in the same way before, but I saw the code of `deprecate` function:
![image](https://user-images.githubusercontent.com/3250203/90487730-7216c080-e16d-11ea-8ebb-0c180cee80b8.png)
It seems that `message_func` would be overwritten by the build-in function. So I use `Deprecated` directly. But anyway, this works, I will change the code.

Reply:
Updated.